tests/Aspire.Azure.AI.OpenAI.Tests/ConformanceTests.cs

```diff
@@ -84,14 +83,18 @@ void ConfigureCredentials(AzureOpenAISettings settings)
         }
     }
 
+    public ConformanceTests(ITestOutputHelper? output = null) : base(output)
```

Comment:
How does this ctor get invoked?

Reply:
When running in the remote process the `output` object is not being passed. Instead the messages get logged to the console, and on the test side we read them as output of that remote process and pipe that to the xunit output.

Reply:
Got it. That makes sense.